examples/README-authz.md

```diff
@@ -0,0 +1,353 @@
+Service accounts can have roles assigned like regular users.
+They can not, however, have groups assigned.
+As a consequence, if you want to target permissions to microservices using service accounts, you can't use Group policies, but are forced to use Role policies.
+Or, thinking about it another way, if you want to limit certain permissions only to regular user accounts where authentication with username and password is required, you should use Group policies, rather than Role policies.
+That's what we see used in `permissions` that start with 'ClusterManager'.
+Performing cluster management is usually done interactively - in person - using CLI tools. 
+It makes sense to require the user to log-in, before using the resulting access token to authenticate to Kafka Broker.
+In this case the access token represents the specific user, rather than the client application.
+
+Before continuing, there is one setting we need to check.
+Due to [a little bug in Keycloak](https://issues.redhat.com/browse/KEYCLOAK-12640) the realm is at this point misconfigured, and we have to fix the configuration manually.
+Under `Clients` / `kafka` / `Authorization` / `Settings` make sure the `Decision Strategy` is set to `Affirmative`, and NOT to `Unanimous`. Click `Save` after fixing it.
+  
+With configuration now in place, let's create some topics, use a producer, a consumer, and try to perform some management operations using different user and service accounts. 
+
+First, we'll spin up a new docker container based on Kafka image previously built by `docker-compose` which we'll use to connect to already running Kafka Broker.
+
+    docker run -ti --rm --name kafka-cli --network docker_default strimzi/example-kafka /bin/sh
+    
+Let's try to produce some messages as client `team-a-client`.
+First, we prepare a configuration file with authentication configuration.
+
+```
+cat > ~/team-a-client.properties << EOF
+security.protocol=SASL_PLAINTEXT
+sasl.mechanism=OAUTHBEARER
+sasl.jaas.config=org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required \
+  oauth.client.id="team-a-client" \
+  oauth.client.secret="team-a-client-secret" \
+  oauth.token.endpoint.uri="http://keycloak:8080/auth/realms/kafka-authz/protocol/openid-connect/token" ;
+sasl.login.callback.handler.class=io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler
+EOF
+```
+
+In Keycloak Console you can find which roles are assigned to `team-a-client` service account, by selecting `team-a-client` under `Clients` section,
+and then opening `Service Account Roles` tab for the client.
+You should see `Dev Team A` realm role assigned. 
+
+We can now use this configuration with Kafka CLI tools.
+
+Make sure the necessary classes are on the classpath:
+
+    export CLASSPATH=/opt/kafka/libs/strimzi/*:$CLASSPATH
+
+Now let's try produce some messages to topic 'my-topic':
```

Comment:
Starting from here, it seems that text is in "italics"

Reply:
Looks good [here](https://github.com/mstruk/strimzi-kafka-oauth/blob/authz/examples/README-authz.md).